src/commands/deploy/command.ts

```diff
@@ -38,6 +45,11 @@ function decideDeployType(): DeployType {
         return DeployType.NextJS;
     }
 
+    // Check if genezio.config.ts exists
+    if (fs.existsSync(path.join(cwd, "genezio.config.ts"))) {
+        return DeployType.GenezioConfigTs;
+    }
+
```

Comment:
I think this should be moved down as it may interfere with Next.js deployment if the Next.js project happens to have a `genezio.config.ts` file

Reply:
I agree, I think I will do this in a future PR where I will specifically test a Nextjs deployment with `genezio.config.ts`.